# PcFeatures for a single cluster

Header header

PcBoundingBox[] boxes
int32 count
